projects/prison-case-notes-to-probation/src/main/resources/application.yml

```diff
@@ -30,6 +30,7 @@ spring.security.oauth2.client:
   provider:
     hmpps-auth:
 
+sentry.debug: true
```

Comment:
We should let Feign throw an error on 404 rather than try and deal with it ourselves.  Currently Feign will throw a `NotFound` exception on 404, so yes we can make the response body non-nullable.  I'll give that a try and put a test in for it to make sure it works.